packages/client/src/clients/in-app/store.ts

```diff
@@ -0,0 +1,19 @@
+import { Store } from "@tanstack/store";
+
+import { NetworkStatus } from "../../networkStatus";
+
+import { InAppStoreState } from "./types";
+
+export type InAppStore = Store<
+  InAppStoreState,
+  (cb: InAppStoreState) => InAppStoreState
+>;
+
+export function createStore() {
+  return new Store<InAppStoreState>({
+    networkStatus: NetworkStatus.ready,
+    loading: false,
+    messages: {},
+    queries: {},
```

Comment:
Can i double check / clarify that this is the global store that is meant to store all messages state? If so, i'm a bit confused why it includes `networkStatus`, `loading` and `queries` - aren't they all part of the InAppMessageClient concerns?

Reply:
Yeah, this is the global store. I should move `networkStatus` and `loading` inside of the individual queries. 

I'd lean towards keeping the queries in the top level store in case there are duplicate queries and since it is a bit simpler to have a single store.